Modules/Filtering/DistanceMap/test/itkDirectedHausdorffDistanceImageFilterTest2.cxx

```diff
@@ -49,7 +48,7 @@ itkDirectedHausdorffDistanceImageFilterTest2(int argc, char * argv[])
 
   ITK_TRY_EXPECT_NO_EXCEPTION(reader1->Update());
```

Comment:
This is nor part of #2445, but you could further simplify the tests by using convenience function which was recently added. Example of how to do it: https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/pull/307.

The style in tests:
```cpp
typename ImageType::Pointer image1 = nullptr;
ITK_TRY_EXPECT_NO_EXCEPTION(image1 = itk::ReadImage<ImageType>(argv[1]));
```

Reply:
This could be done in a follow-up PR, and might include using `ReadImage` in many other tests, not just the ones touched by this PR.